src/dos/cdrom.h

```diff
@@ -247,6 +267,7 @@ static  struct imagePlayer {
 	bool	GetCueKeyword(std::string &keyword, std::istream &in);
 	bool	GetCueFrame(int &frames, std::istream &in);
 	bool	GetCueString(std::string &str, std::istream &in);
+	// bool	AddTrack(Track &curr, int &shift, int prestart, int &totalPregap, int currPregap, int frameFromCue);
```

Comment:
Probably not needed.

Reply:
Fixed in a3dc6a407ca75a89293070e24823e33ab1d783c6.